common/src/main/java/org/apache/uniffle/common/config/RssBaseConf.java

```diff
@@ -294,6 +294,18 @@ public class RssBaseConf extends RssConf {
                   + " first combining the username and the password with a colon (uniffle:uniffle123)"
                   + ", and then by encoding the resulting string in base64 (dW5pZmZsZTp1bmlmZmxlMTIz).");
 
+  public static final ConfigOption<String> RSS_STORAGE_WRITE_DATA_BUFFER_SIZE =
+      ConfigOptions.key("rss.storage.write.data.buffer.size")
```

Comment:
The last segment prefer using Camel style like `rss.storage.write.dataBufferSize`.

Reply:
I have changed the code as you suggested.